build/examples.js

```diff
@@ -3,12 +3,6 @@ var path = require("path");
 module.exports = {
     mouseposition: path.join(__dirname, "..", "web", "client", "examples", "mouseposition", "app"),
     scalebar: path.join(__dirname, "..", "web", "client", "examples", "scalebar", "app"),
-    "3dviewer": path.join(__dirname, "..", "web", "client", "examples", "3dviewer", "app"),
     featuregrid: path.join(__dirname, "..", "web", "client", "examples", "featuregrid", "app"),
-    print: path.join(__dirname, "..", "web", "client", "examples", "print", "app"),
-    login: path.join(__dirname, "..", "web", "client", "examples", "login", "app"),
-    plugins: path.join(__dirname, "..", "web", "client", "examples", "plugins", "app"),
-    api: path.join(__dirname, "..", "web", "client", "examples", "api", "app"),
-    hooks: path.join(__dirname, "..", "web", "client", "examples", "hooks", "app"),
-    lazyplugins: path.join(__dirname, "..", "web", "client", "examples", "lazyplugins", "app")
```

Comment:
Probably we could remove `lazyplugins` and `hooks` from this index but should we remove also the related code? It's a new feature and we haven't so much reference to an App with lazy plugins or hooks but probably I'm wrong

Reply:
I'm not sure they are maintained so I think we should provide an alternative and more powerful set of examples when framework has been revised, with something that is more useful for developers